extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSFilter.java

```diff
@@ -55,6 +55,10 @@ public static boolean isConfiguredWithWildcard(Optional<List<String>> optionalLi
         return list.isEmpty() || (list.size() == 1 && "*".equals(list.get(0)));
     }
 
+    private static boolean isOriginConfiguredWithWildcard(List<String> origins) {
+        return origins.size() == 1 && "*".equals(origins.get(0));
```

Comment:
Why is this being evaluated on every request? It could just be a boolean.

Reply:
@stuartwdouglas Should be better now